stream-transformation-test/sample-transformations/pom.xml

```diff
@@ -22,6 +24,14 @@
         <dependency>
             <groupId>uk.gov.justice.utils</groupId>
             <artifactId>test-utils-core</artifactId>
+            <version>${test.utils.core.version}</version>
+            <exclusions>
```

Comment:
Why do we need the exclusion? Is the test utils using a different version? we should probably try and make it consistent across all of them by updating the common bom perhaps.